src/widget/wlibrary.cpp

```diff
@@ -52,14 +52,9 @@ void WLibrary::switchToView(const QString& name) {
     QMutexLocker lock(&m_mutex);
     //qDebug() << "WLibrary::switchToView" << name;
 
-    WTrackTableView* ttView = qobject_cast<WTrackTableView*>(
+    LibraryView* oldLibraryView = dynamic_cast<LibraryView*>(
```

Comment:
qobject_cast should work and is much faster.

Reply:
LibraryView is not a qobject but a mixin. I always try qobject_cast first :)

Reply:
OK